src/App.vue

```diff
@@ -346,17 +368,6 @@ export default class App extends Mixins(AuthApiMixin, CommonMixin, DateMixin, Fi
     await this.updateCurrentJsDate()
     this.updateCurrentJsDateId = setInterval(this.updateCurrentJsDate, 60000)
 
-    // add handler to prompt user if there are changes, before unloading this page
-    window.onbeforeunload = (event: any) => {
-      if (this.haveUnsavedChanges || this.isEditing) {
-        // cancel closing the page
-        event.preventDefault()
-        // pop up confirmation dialog
-        // NB: custom text is not supported in all browsers
-        event.returnValue = 'You have unsaved changes. Are you sure you want to leave?'
-      }
-    }
```

Comment:
This is to prevent navigating away from this page (eg, direct URL entry) or closing the current tab/window. Why remove it?

Reply:
This would cause a double confirmation since we are using the dialog to confirm by leaving by hitting 'Cancel'. 
I should probably refactor and toggle one of the conditions (`haveUnsavedChanges`) here when leaving, in order to keep this in.

Reply:
Hmm, ok i had that code in to set the flag to false. 
Will have to look at it closer as to why it was still calling this even after the confirmation dialog.

Reply:
I must have been smoking something, or perhaps i added in the flag setting after the fact. 
Either way, good catch, thank you!

Fixed